docs/reference/commandline/swarm_update.md

```diff
@@ -12,15 +12,16 @@ parent = "smn_cli"
 
 # swarm update
 
-	Usage:    docker swarm update [OPTIONS]
-
-	update the Swarm.
-
-	Options:
-	      --auto-accept value   Acceptance policy (default [worker,manager])
-	      --help                Print usage
-	      --secret string       Set secret value needed to accept nodes into cluster
-
+    Usage:  docker swarm update [OPTIONS]
+    
+    update the Swarm.
+    
+    Options:
+          --auto-accept value                      Auto acceptance policy (worker, manager or none)
+          --dispatcher-heartbeat-period duration   Dispatcher heartbeat period (default 5s)
```

Comment:
Unrelated to this PR, but should the flag just be called `--dispatcher-heartbeat`?

Or `--worker-heartbeat`, `--node-heartbeat` ...

Although that can get confusing since Raft peers have a different HB.

/cc @LK4D4 @aaronlehmann